tcms/testcases/forms.py

```diff
@@ -104,7 +104,24 @@ def populate(self, product_id=None):
             self.fields['category'].queryset = Category.objects.all()
 
 
-class NewCaseForm(BaseCaseForm):
+class NewCaseForm(BaseCaseForm, forms.ModelForm):
+
+    class Meta:
+        model = TestCase
+        fields = ['summary',
+                  'default_tester',
+                  'requirement',
+                  'is_automated',
+                  'script',
+                  'arguments',
+                  'extra_link',
+                  'case_status',
+                  'priority',
+                  'product',
+                  'category',
+                  'notes',
+                  'text']
```

Comment:
Are these all fields in the model ? Looks like it. You shouldn't need to specify all fields by hand if all of them will be edited.

ATM I'm not sure how well this will play with the email notification settings.